dm/dm-webui-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 使用 WebUI 管理 DM 迁移任务
+summary: 学习如何使用 WebUI 来方便的管理数据迁移任务
+---
+
+# 使用 WebUI 管理 DM 迁移任务
+
+DM WebUI 是为了方便大量迁移任务的管理、简化操作步骤的目的开发，其主要特性包含运行状态监控，上游数据源配置，以及迁移任务管理等。DM WebUI 可以从任意 master 节点访问，其端口与 OpenAPI 保持一致，默认为 8261。访问地址示例：`http://{master_ip}:{master_port}/dashboard/`。
+
+> **注意：**
+>
+> - DM WebUI 当前为实验特性，尚不建议用于生产环境。
+> 
+> - DM WebUI 中 task 的生命周期有所改变，不建议与 dmctl 同时使用。
+
+## 实验特性开关
+
+若要使用 DM WebUI，需要在 master 配置中开启`openapi`配置：
+
+```
+openapi = true
+```
+
+## Dashboard
+
+Dashboard 是内嵌了 DM 的 Grafana Dashboard，包含两个视图`Standard`和`Professional`分别展示了不同角度的监控信息。
+
+如果要在 DM WebUI 中正确显示 Dashboard，首次部署必须确认以下两个步骤正确完成：
+
+1. 部署时正确安装监控相关组件：`monitoring_servers`和`grafana_servers` ，其中`grafana_servers`的配置必须如下所示：
+
+    ```
+    grafana_servers:
+      - host: 10.0.1.14
+        # port: 3000
+        # deploy_dir: /tidb-deploy/grafana-3000
+        config:
+          auth.anonymous.enabled: true
+          security.allow_embedding: true
+    ```
+
+2. 若`grafana_servers`使用了非默认的 IP 和端口，则需要在 WebUI 的 Dashboard 界面填写正确的 IP 和端口。
```

Comment:
For a dm cluster that has been upgraded to the latest version, how do DM know what the default port is?